packages/vue/src/generators/library/library.ts

```diff
@@ -86,14 +120,23 @@ export async function libraryGeneratorInternal(tree: Tree, schema: Schema) {
     });
   }
 
-  if (options.publishable || options.bundler !== 'none') {
+  if (
+    !options.isUsingTsSolutionConfig &&
+    (options.publishable || options.bundler !== 'none')
+  ) {
     updateJson(tree, `${options.projectRoot}/package.json`, (json) => {
       json.name = options.importPath;
       return json;
     });
   }
 
-  if (!options.skipTsConfig) {
+  if (options.bundler === 'none' && options.addPlugin) {
```

Comment:
It would be weird to happen, but `addPlugin` can be `false` and the workspace could still have the plugin registered from a previous generation. So, we should always ensure the project is excluded. The helper will ignore if the plugin is not registered.
```suggestion
  if (options.bundler === 'none') {
```

Reply:
This still needs fixing.